erpnext/selling/doctype/customer/customer.js

```diff
@@ -237,6 +237,11 @@ frappe.ui.form.on("Customer", {
 		if(frm.days_selected) {
 			frm.set_value("delivery_days", JSON.stringify(frm.days_selected.get_value()));
 		}
+		frappe.db.exists('Customer', frm.doc.customer_name).then( exists => {
+			if (exists) {
+				frappe.msgprint(__(`Customer ${frm.doc.customer_name} already exists.`));
+			}
+		});
```

Comment:
Make customer_name unique in the Doctype itself. BTW @DeveloperMetal @hrwX @neilLasrado will this change break  JHA?

Reply:
It could, we will have to run a test and check.

Reply:
@MyuddinKhatri @neilLasrado @DeveloperMetal This change will break JHA since there are duplicate customer_names in the system